### PR TITLE
Add tags support for users

### PR DESCRIPTION
**Pull request by bh42**

### CODE REVIEW

**Comment by blakmatrix** [comment]
👍

**Comment by cryptomail** [comment]
Ouch we're getting some namespace problems here....This file references Tickets... did you mean Users?

File: lib/client/users.js [hunk 1]
```diff
@@ -131,3 +131,26 @@ Users.prototype.incremental = function (startTime, cb) {
 Users.prototype.incrementalSample = function (startTime, cb) {
   return this.request('GET', ['incremental', 'users', 'sample', { start_time: startTime }], cb);
 };
+
+// ######################################################## User Tags
+
+
+// ====================================== List Tags of User
+Tickets.prototype.listTags = function (userId, tags, cb) {
```
Comment (cryptomail) [inline lib/client/users.js:139]: Users?....This is currently distributed... :(

Comment (cryptomail) [reply lib/client/users.js:139]: Fixed in subsequent [PR](https://github.com/blakmatrix/node-zendesk/pull/358) !  It works though :) 
Also extraneous param.
Screenshot 2023-08-18 at 9 37 08 AM

Comment (blakmatrix) [reply lib/client/users.js:139]: Thanks a bunch for catching this, You're Awesome!

Comment (cryptomail) [reply lib/client/users.js:139]: NP!  Keeping the magic alive!  New dist working great!

**Pull request merged**